dev/PowerNotifications/PowerNotifications.cpp

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <future>
+#include <PowerNotifications.h>
+#include <frameworkudk\PowerNotificationsPal.h>
+#include <PowerManager.g.cpp>
+#include <powrprof.h>
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+
+    // EnergySaverStatus Functions
+    EventType& EnergySaverStatus_Event()
```

Comment:
I can't use a const due to mismatches in later code.

Also is there an advantage of using a getter?

Reply:
I thought this was a method of the object. If it's a free floating function then ignore my comment it's not applicable.

If it's a method of an object then declaring it as a getter gives more natural property-like syntax (reasonable for simple property-like things). If it's a free-floating function then making the method const is N/A since methods of objects can be const but non-object functions can't.